packages/venia-ui/lib/components/ProductImageCarousel/thumbnail.js

```diff
@@ -44,24 +47,26 @@ const Thumbnail = props => {
     const isDesktop = windowSize.innerWidth >= 1024;
 
     const thumbnailImage = useMemo(() => {
-        const src = file
-            ? resourceUrl(file, {
-                  type: 'image-product',
-                  width: DEFAULT_THUMBNAIL_WIDTH,
-                  height: DEFAULT_THUMBNAIL_HEIGHT
-              })
-            : transparentPlaceholder;
+        if (!isDesktop) {
+            return null;
+        }
 
-        return isDesktop ? (
+        return Boolean(file) ? (
```

Comment:
```suggestion
        return file ? (
```

Pretty sure the ternary makes this redundant.

Reply:
Resolved in https://github.com/magento/pwa-studio/pull/1871/commits/051efa8201892481f3d044336efcd92705b38c75